src/test/java/com/solvd/micro9/users/service/EsUserCommandHandlerTest.java

```diff
@@ -0,0 +1,153 @@
+package com.solvd.micro9.users.service;
+
+import com.google.gson.Gson;
+import com.solvd.micro9.users.domain.aggregate.User;
+import com.solvd.micro9.users.domain.command.CompleteTransactionCommand;
+import com.solvd.micro9.users.domain.command.CreateUserCommand;
+import com.solvd.micro9.users.domain.command.DeleteUserCommand;
+import com.solvd.micro9.users.domain.es.Es;
+import com.solvd.micro9.users.domain.es.EsStatus;
+import com.solvd.micro9.users.domain.es.EsType;
+import com.solvd.micro9.users.domain.es.EsUser;
+import com.solvd.micro9.users.messaging.KfProducer;
+import com.solvd.micro9.users.persistence.eventstore.EsUserRepository;
+import com.solvd.micro9.users.service.cache.RedisConfig;
+import com.solvd.micro9.users.service.impl.EsUserCommandHandlerImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.redis.core.ReactiveHashOperations;
+import org.springframework.data.redis.core.ReactiveRedisOperations;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
@eyakauleva be careful with sharing test state

Reply:
fixed 1219506